### ISSUE TITLE
Subtracting bias in photospline/I3SplineTable even if derivatives are available (Trac #2236)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2228]
After evaluating photosplines a bias must only be subtracted if no derivatives are available but it is always done.
I am going to commit a fix for that soon.

Migrated from https://code.icecube.wisc.edu/ticket/2236
```json
{
    "status": "closed", 
    "changetime": "2019-02-04T17:52:51", 
    "description": "After evaluating photosplines a bias must only be subtracted if no derivatives are available but it is always done.\nI am going to commit a fix for that soon.", 
    "reporter": "mhieronymus", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1549302771596052", 
    "component": "combo reconstruction", 
    "summary": "Subtracting bias in photospline/I3SplineTable even if derivatives are available", 
    "priority": "normal", 
    "keywords": "Bug I3SplineTable", 
    "time": "2019-01-31T08:17:04", 
    "milestone": "", 
    "owner": "mhieronymus", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20190131T08:29:41: 
* mhieronymus commented:

> In changeset 171057:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="171057"
> Fix #2228 Subtract bias only if derivatives are involved.
> ```


* mhieronymus changed status from "new" to "closed"
* mhieronymus changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190204T17:52:51: nwhitehorn commented:

> In changeset 171112:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="171112"
> Fix #2228 Subtract bias only if derivatives are involved.
> ```
